xGame/Java/xgame/src/main/java/com/xgame/restservice/controller/ChessMatchController.java

```diff
@@ -14,11 +19,21 @@
 	@Autowired
 	IChessMatchService matchService;
 	
+	@GetMapping("/match/invite")
+	public List<MatchInviteViewModel> getInvites(@RequestParam(value = "playerId") int playerId) {
+		return matchService.getInvites(playerId);
+	}
+	
 	@PostMapping("/match")
 	public MatchViewModel createMatch(
 			@RequestParam(value = "whiteId") int whiteId,
 			@RequestParam(value = "blackId") int blackId) {
 		
 		return matchService.createMatch(whiteId, blackId);
 	}
+	
+	@PatchMapping("/match/invite")
```

Comment:
What will be the route for declining an invitation? Since they're both patches, this might have to be more specific

Reply:
yeah probably. we'll fix it when we implement the declining functionality.

Reply:
Might be worth changing now so that we don't have to change the route in 2 places (here and in the React) when we add declining